web3/eth.py

```diff
@@ -262,6 +270,14 @@ def sign(self, account, data):
             "eth_sign", [account, data_hash],
         )
 
+    def verify(self, account, data, signature):
+        eth_data_hash = self.getEthSignHash(data)
+        signer = self._extract_ecdsa_signer(force_bytes(eth_data_hash), force_bytes(signature))
+        return signer == account
+
+    def getEthSignHash(self, msg):
+        return self.web3.sha3('\x19Ethereum Signed Message:\n' + str(len(msg)) + msg, encoding="utf8")
```

Comment:
Can you remove the hashing from this and change it to be something like `prefixMessageForSigning`, and then do the hashing outside of this function?